examples/async/async.bal

```diff
@@ -4,7 +4,16 @@ import ballerina/runtime;
 
 int count = 0;
 
-http:Client clientEndpoint = new("https://postman-echo.com");
+http:ClientConfiguration endpointConfig = {
+    secureSocket: {
+        trustStore: {
+            path: "${ballerina.home}/bre/security/ballerinaTruststore.p12",
+            password: "ballerina"
+        }
+    }
+};
```

Comment:
We don't need this here. We can use "http://postman-echo.com". Will send the changes in a separate PR